.github/workflows/ci.yml

```diff
@@ -100,7 +99,7 @@ jobs:
           toolchain: ${{ env.MINIMUM_SUPPORTED_RUST_VERSION }}
           components: clippy
 
-      # We don't currently have features, so there's no need to run these checks
+      # Because we re-export kurbo features --no-default-features alone is not a valid set features.
```

Comment:
Do we instead need to check that we still work with only the libm feature?

Reply:
Probably, perhaps another option might be to: `cargo hack --exclude-no-default-features --feature-powerset`
This could reduce the number amount of stuff in ci.yml, but that requires bringing cargo hack into the ci, any preferences?